OpenIPSL/Electrical/Controls/PSSE/TG/HYGOV.mo

```diff
@@ -71,9 +72,9 @@ model HYGOV "HYGOV - Hydro Turbine-Governor model"
     T=T_r,
     y_start=0)
     annotation (Placement(transformation(extent={{-106,0},{-94,12}})));
-protected
-  parameter Real h0=1 "water head initial value";
+  // parameter Real h0=1 "water head initial value";
```

Comment:
I would also remove the out commented lines here.

Reply:
Yes, agree.

Reply:
Done in [f4776b7](https://github.com/OpenIPSL/OpenIPSL/pull/309/commits/f4776b7079929e50ed673f7d4392233a20f224a7), could you review @GiuseppeLaera or @marcelofcastro